packages/components/pie-button/src/index.ts

```diff
@@ -207,23 +224,60 @@ export class PieButton extends FormControlMixin(LitElement) implements ButtonPro
         }
 
         return html`
-                    <pie-spinner
-                        size="${spinnerSize}"
-                        variant="${spinnerVariant}">
-                    </pie-spinner>`;
+            <pie-spinner
+                size="${spinnerSize}"
+                variant="${spinnerVariant}">
+            </pie-spinner>`;
+    }
+
+    renderAnchor (classes: ClassInfo) {
+        const {
+            href, iconPlacement, rel, target,
+        } = this;
+
+        return html`
+            <a
+                href="${ifDefined(href)}"
+                rel="${ifDefined(rel)}"
+                target="${ifDefined(target)}"
+                class="${classMap(classes)}">
+                ${iconPlacement === 'leading' ? html`<slot name="icon"></slot>` : nothing}
+                <slot></slot>
+                ${iconPlacement === 'trailing' ? html`<slot name="icon"></slot>` : nothing}
```

Comment:
I've been considering this for a while, and I'm becoming less convinced that using a slot for button text is the best approach, especially when it comes to link buttons.

While I understand that the convention is to use a slot, I can't think of many situations where we would want anything other than plain text - particularly for the link variant.

What do you think?

Reply:
Yes I think I agree. As you said to me this might be useful for something like `nuxt-link`, etc, but there might be other workaround for that like adding more possible values for "tag".

Maybe we could suggest this as a change when we move to v1?

Reply:
We discussed this on Slack and we're going to raise this as a separate discussion with the team.